Formula/csound.rb

```diff
@@ -5,7 +5,7 @@ class Csound < Formula
       tag:      "6.15.0",
       revision: "18c2c7897425f462b9a7743cee157cb410c88198"
   license "LGPL-2.1-or-later"
-  revision 4
+  revision 5
```

Comment:
This change isn't really relevant for users who already have it installed, so we can skip the revision bump here.